api/src/server.rs

```diff
@@ -340,641 +339,15 @@ where
     }
 }
 
-fn decode_event_id(value: &str) -> Result<EventId, ApiError> {
+pub(crate) fn decode_event_id(value: &str) -> Result<EventId, ApiError> {
     multibase::decode(value)
         .map_err(|err| ApiError(format!("multibase error: {err}")))?
         .1
         .try_into()
         .map_err(|err| ApiError(format!("invalid event id: {err}")))
 }
-fn decode_event_data(value: &str) -> Result<Vec<u8>, ApiError> {
+pub(crate) fn decode_event_data(value: &str) -> Result<Vec<u8>, ApiError> {
```

Comment:
and this one?

Reply:
This is being used in the test file as well as the BuildResponse